obspy/core/trace.py

```diff
@@ -2779,6 +3241,207 @@ def _data_sanity_checks(value):
                "arrays are allowed for initialization.") % str(value.shape)
         raise ValueError(msg)
 
+class FrequencyDomainTrace(BaseTrace):
+
+    def ifft(self):
+        tr = Trace()
+        tr.stats = Stats()
+        tr.stats.starttime = self.stats.starttime
+        tr.data = fftpack.irfft(self.data)
```

Comment:
Well, yeah, our array would be one item longer (imaginary part ==0 at zero frequency) than the one we get, so there's no chance to avoid a copy by just setting a different type or other tricks..